src/Core/src/Configuration.php

```diff
@@ -12,7 +12,7 @@
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-final class Configuration
+class Configuration
```

Comment:
Hm.. Na. I think I was happy with this as final.

Reply:
Reverted